CMakeLists.txt

```
@@ -60,45 +54,43 @@ add_library(${PROJECT_BASE_NAME} SHARED ${library_sources} )
 
```

I just realised that the library also needs the major version in the name (for the same reason the server needs it). In addition, to keep it simple it should have the full server name so a generic doocs and epics server can be installed at the same time . Otherwise we would have to introduce a library package which is used by all flavours, which would be overkill.
```suggestion
add_library(${PROJECT_NAME}-impl SHARED ${library_sources} )
```
I added the ``-impl`` suffic because the target needs a different name. I could have taken ``-lib``, but did not like the resulting file name ``libdoocs-generic-chimeratk-server-lib.so``

---

Done